cli/dcoscli/data/help/package.txt

```diff
@@ -91,3 +92,6 @@ Positional Arguments:
     <repo-url>
         URL of the package repository. For example,
         https://universe.mesosphere.com/repo.
+    <repo-file>
+        A file containing package repositories. JSON files are acceptable,
+        including the output of `dcos package list --json`.
```

Comment:
We should explain the format of this. Something like: 

A file containing package repositories listed in the format of `dcos package list --json`. Example: { "repositories": [ {"name": "Universe", "uri": "uri-here"}]}